draft-ietf-ppm-dap.md

```diff
@@ -2029,21 +2014,18 @@ variant {
 If `state == Continued(prep_state)`, then the Helper stores `state` to
 prepare for the next continuation step ({{aggregation-helper-continuation}}).
 
-If `state == Finished(out_share)`, the Helper MUST resolve replay of the
-report. It does so by checking if the report ID was previously aggregated for
-this task. If so, it responds with
+If `state == Finished(out_share)`, then the Helper commits to `out_share` as
+described in {{batch-buckets}}. If commitment fails with error `commit_error`,
+then instead of the `continue` response above, the Helper responds with
 
 ~~~ tls-presentation
 variant {
   ReportID report_id;
-  PrepareRespState prepare_resp_state = reject;
-  Reporterror report_error = report_replayed;
+  PrepareRespstate prepare_resp_state = reject;
+  ReportError = commit_error;
```

Comment:
Syntax looks inconsistent with what we had before. Should it be `ReportError report_error = commit_error`?

Likewise below.